### PR TITLE
Allow images without entrypoint with -e

### PR DESCRIPTION
**Pull request by DiegoPomares**
Currently the only way around this is to create yaml file manually.

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -52,8 +52,8 @@ var installCommand = &cobra.Command{
 			return err
 		}
 		
-		if imageInspect.Config.Entrypoint == nil {
-			return fmt.Errorf("the image '%s' is not compatible with Whalebrew: it does not have an entrypoint", imageName)
+		if imageInspect.Config.Entrypoint == nil && customEntrypoint == "" {
+			return fmt.Errorf("The image '%s' does not have an entrypoint, please provide a custom entrypoint", imageName)
```
Comment (tjamet) [inline cmd/install.go:56]: Shall we inform how to do so?
```suggestion
			return fmt.Errorf("The image '%s' does not have an entrypoint, please provide a custom entrypoint adding '--entrypoint my-program'", imageName)
```

**Comment by tjamet** [comment]
Awesome contrib! thanks!
Indeed, `--entrypoint` is available and it makes total sense to handle it even when there is no entrypoint in the image.
Tell me your thoughts on the suggestion and I will proceed to merging this PR

**Review by tjamet — APPROVED** [review]

**Comment by DiegoPomares** [comment]
Implemented your suggestion :)

**Review by tjamet — APPROVED** [review]

**Comment by tjamet** [comment]
thanks! va dentro!

**Pull request merged**